resolve_conflicts.py

```
@@ -0,0 +1,206 @@
+        elif line.startswith("=======") and start_idx != -1:
+            middle_idx = i
+        elif line.startswith(">>>>>>>") and start_idx != -1 and middle_idx != -1:
+            conflicts.append((start_idx, middle_idx, i))
+            start_idx = -1
+            middle_idx = -1
+
+    return conflicts
+
+
+# Resolve a specific conflict
+def resolve_conflict(content: str, conflict: Tuple[int, int, int]) -> str:
```

Ruff (0.11.9)

`resolve_conflict` is too complex (15 > 10)

(C901)